contributors/devel/development.md

```diff
@@ -247,6 +247,8 @@ _If you have upstream write access_, please refrain from using the GitHub UI for
 creating PRs, because GitHub will create the PR branch inside the main
 repository rather than inside your fork.
 
+When pushing commits to a pull request, don't force push. Force pushing can corrupt your pull request.
```

Comment:
I was always doing the following for my updated PR

```
1. Update code
2. git add .
3. git commit --amend
4. git push origin <mybranch> --force
```

And I can always get my PR uploaded successfully, anything wrong with my process?